cpp/src/bindings.cpp

```diff
@@ -396,6 +419,76 @@ PYBIND11_MODULE(_pypowsybl, m) {
+                return (double) p.low_voltage_absolute_threshold;
+            }, [](security_analysis_parameters& p, const double lowVoltageAbsoluteThreshold) {
+                p.low_voltage_absolute_threshold = lowVoltageAbsoluteThreshold;
+            })
+            .def_property("high_voltage_proportional_threshold", [](const security_analysis_parameters& p) {
+                return (double) p.high_voltage_proportional_threshold;
+            }, [](security_analysis_parameters& p, const double highVoltageProportionalThreshold) {
+                p.high_voltage_proportional_threshold = highVoltageProportionalThreshold;
+            })
+            .def_property("high_voltage_absolute_threshold", [](const security_analysis_parameters& p) {
+                return (double) p.high_voltage_absolute_threshold;
+            }, [](security_analysis_parameters& p, const double highVoltageAbsoluteThreshold) {
+                p.high_voltage_absolute_threshold = highVoltageAbsoluteThreshold;
+            })
+            .def_property("provider_parameters_keys", [](const security_analysis_parameters& p) {
+                return std::vector<std::string>(p.provider_parameters_keys, p.provider_parameters_keys + p.provider_parameters_keys_count);
+            }, [](security_analysis_parameters& p, const std::vector<std::string>& providerParametersKeys) {
+                pypowsybl::deleteCharPtrPtr(p.provider_parameters_keys, p.provider_parameters_keys_count);
+                p.provider_parameters_keys = pypowsybl::copyVectorStringToCharPtrPtr(providerParametersKeys);
+                p.provider_parameters_keys_count = providerParametersKeys.size();
+            })
+            .def_property("provider_parameters_values", [](const security_analysis_parameters& p) {
+                return std::vector<std::string>(p.provider_parameters_values, p.provider_parameters_values + p.provider_parameters_values_count);
+            }, [](security_analysis_parameters& p, const std::vector<std::string>& providerParametersValues) {
+                pypowsybl::deleteCharPtrPtr(p.provider_parameters_values, p.provider_parameters_values_count);
+                p.provider_parameters_values = pypowsybl::copyVectorStringToCharPtrPtr(providerParametersValues);
+                p.provider_parameters_values_count = providerParametersValues.size();
+            });
+    
+     py::class_<sensitivity_analysis_parameters, std::shared_ptr<sensitivity_analysis_parameters>>(m, "SensitivityAnalysisParameters")
+            .def(py::init(&initSensitivityAnalysisParameters))
+            .def_property("load_flow_parameters", [](const sensitivity_analysis_parameters& p) {
+                return (load_flow_parameters_struct) p.as_load_flow_parameters;
+            }, [](sensitivity_analysis_parameters& p, const load_flow_parameters loadFlowParameters) {
+                p.as_load_flow_parameters = loadFlowParameters;
```

Comment:
like for security analysis, we need to deep-copy the loadflow parameters, to avoid double-free of memory